### DISCUSSION TITLE
`Assertion tri_idx < m_indices.size()' failed.`

### DISCUSSION THREAD

**Discussion by frutas-fruit** [xissburg/edyn#91] [General]

**Comment by xissburg** [comment]
Maybe you could try deleting the cached binary files and let them be regenerated from the obj?

**Comment by xissburg** [comment]
Can you try checking what are the contents of `m_indices`? Is it empty?

**Reply by xissburg** [reply]
I mean in the debugger at the time of crash.

**Reply by xissburg** [reply]
I am wondering whether the triangle mesh is being initialized at all. In that case, `m_indices` would be empty.

**Reply by xissburg** [reply]
The code looks correct. Does it crash on start up or when a collision is first detected? Is it loading from obj or was the binary generated?
